tests/DoctrineTest/InstantiatorTest/InstantiatorTest.php

```diff
@@ -104,6 +112,8 @@ public function testInstancesAreNotCloned() : void
      * Provides a list of instantiable classes (existing)
      *
      * @return string[][]
+     *
+     * @phpstan-return class-string[][]
```

Comment:
```suggestion
     * @phpstan-return list<array{class-string}>
```

Reply:
Thanks, I'll learn one day 😅 , so `list` when indexes don't matter. Why not `list<list<class-string>>`?

Reply:
Yes, the inner one should match the signature (not checked by SA tools AFAIK, but might come in the future). What SA tools do help with is adding a new test case in the provider: they will complain if you add an extra item in the array for instance.